src/patternfly/components/AppLauncher/app-launcher.scss

```diff
@@ -189,24 +191,32 @@
   transform: var(--pf-c-app-launcher__menu-item-external-icon--Transform); // move icon a bit to appear more visually centered
 }
 
-// remove this block at breaking change period
 .pf-c-app-launcher__separator {
   height: var(--pf-c-app-launcher__separator--Height);
   margin-top: var(--pf-c-app-launcher__separator--MarginTop);
   margin-bottom: var(--pf-c-app-launcher__separator--MarginBottom);
   background-color: var(--pf-c-app-launcher__separator--BackgroundColor);
   border: 0; // for use when separator is an <hr>
+
+  &:last-child {
+    margin-bottom: 0;
```

Comment:
should you `--pf-c-app-launcher__separator--MarginBottom: 0` as it keeps specificity of `margin-bottom` at 10?